tutorials-dp0-3/portal-dp0-3-3.rst

```diff
@@ -0,0 +1,421 @@
+Then add a pair of new plots, clicking on the "plot settings."  
+In both cases, enter ``midPointMjdTai - 60000`` for X-axis, but for Y axis - enter ``topocentricDist`` for one plot, and ``heliocentricDist`` for the other.  
+After you remove the panel containing the plot made in the previous step, you will see the plots as below.  
+
+.. figure:: /_static/portal_tut03_step04b.png
+    :width: 600
+    :name: portal_tut03_step04b
+    :alt: A screenshot of two plots showing the heliocentric and topocentric distance of the trans-Neptunian object as a function of time.
+
+    Heliocentric and topocentric distance of the TNO as a function of time.  
+Note the periodic change of the topocentric distance with time resulting from the Earth's motion around the Sun - a different view of the same effect you saw in Step 3.5.  
+
+.. _DP0-3-Portal-3-Step-5:
+
+Step 5. Visualize in 3-D the trajectory of the TNO by viewing the 2-D projection of its orbit
+=========================================================================================
+
+5.1.  Navigate again to the ADQL query interface.  
+Execute the query below to extract the helio- and topocentric distances of the TNO - so you can visualize its trajectory:  
+
+.. code-block:: SQL 
+
+    SELECT heliocentricX, heliocentricY, heliocentricZ,
+    topocentricX, topocentricY, topocentricZ, ssObjectId
+    FROM dp03_catalogs_10yr.SSSource
+    WHERE ssObjectId = -735085100561880491
+
+
+5.2.  Plot the heliocentric Z distance as a function of heliocentic X distance by clicking on the "plot setings" icon and selecting ``heliocenticZ`` for y and ``heliocentricX`` for x.  
+Note that the object moves slowly in heliocentric coordinate X as well as in Y (by a comparison to, e.g., Earth's motion), covering only a few au in 10 years.  
+This is expected given its multi-au distance from the Sun.  
+Now observe that the object's trajectory is not constant in Z - and that means that its orbit is not in the plane of the Ecliptic during the simunated Rubin observation, but the object does pass through the ecliptic plane when Z = 0.  
+Next, plot the ``topocentricX`` vs. ``heliocentricX`` where you can clearly see the effect of position of the TNO on the sky as a result of Earth's orbital motion.  
+
+ .. figure:: /_static/portal_tut03_step05a.png
+    :name: portal_tut03_step05a
+    :width: 600
+    :alt: A screenshot of two plots showing the heliocentric and topocentric distance of the trans-Neptunian object as a function of time.
+
+    Visualization of the 3-D TNO's trajectory by viewing the 2-D projections of its trajectory.  
+
+.. **FIND MORE INTERESTING THINGS TO DO AND EXPLORE WITH THIS TNO!**
+
+.. **PLOT DISTANCES OVER TIME, OR MAYBE GET THE HELIO XYZ AND PLOT OUT ORBITAL ARCS, ETC.**
+
+.. **CONSULT WITH ANDRES WHO IS WORKING ON A TNO NB**
+
+
+
+.. _DP0-3-Portal-3-Step-6:
+
+Step 6.  Exercises for the learner: 
```

Comment:
Do not end section heading with colon (or any other punctuation) this doesn't look right in ToC

Reply:
ok